### PR TITLE
Jacoco Coverage reports

### PR DESCRIPTION
**Pull request by wmbest2**
@vokal/web

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "cobertura-parse",
```
Comment (wmbest2) [inline package.json]: Oops gotta fix this file

**Comment by scottferg** [comment]
LGTM. Even matches our js coding standards.

File: source/index.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        {
+            var fullPath = pack.$.name + '/' + s.$.name
+            var className = fullPath.substring(0, fullPath.lastIndexOf('.'));
+
+            var c;
+            pack.class.forEach( function( cl ) {
+                if (cl.$.name == className) {
+                    c = cl;
+                }
+            });
```
Comment (jrit) [inline source/index.js]: Using `forEach` to search for a matching value like this is a little odd. Maybe use `filter` to find a match and then use the first result instead?

File: source/index.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                    found: c.method ? c.method.length : 0,
+                    hit: 0,
+                    details: !c.method ? [] : c.method.map( function ( m )
+                    {
+                        var hit = false;
+                        m.counter.forEach( function (counter) {
+                            if (counter.$.type == "METHOD" && counter.$.covered == "1") {
+                                hit = true;
+                            }
+                        });
```
Comment (jrit) [inline source/index.js:37]: Maybe use `some` instead of `forEach` here?

File: source/index.js (hunk 2 above)
Comment (wmbest2) [inline source/index.js]: will do

**Comment by jrit** [comment]
Just the 2 array suggestions, then LGTM. Think you need to activate it on Travis too.

**Comment by wmbest2** [comment]
@jrit How is that?

**Comment by jrit** [comment]
Looks good. You can omit those args if you want, it won't change anything, just a little cleaner read without them.

**Comment by wmbest2** [comment]
I removed them if someone wants to merge.

**Pull request merged**